src/components/EpisodeCard.tsx

```diff
@@ -0,0 +1,38 @@
+import IEpisode from "../interfaces/IEpisode";
+import cleanString from "../util/cleanString";
+import padNumber from "../util/padNumber";
+import "./episodeCard.css";
+
+interface EpisodeCardProps {
+    key: number;
+    episode: IEpisode;
+}
+
+export function EpisodeCard(props: EpisodeCardProps): JSX.Element {
+    const e = props.episode;
```

Comment:
This is not easy to read/understand and could be avoided with destructuring props.